pkg/hintrunner/hint_test.go

```diff
@@ -231,3 +234,38 @@ func TestWideMul128IncorrectRange(t *testing.T) {
 	err := hint.Execute(vm)
 	require.ErrorContains(t, err, "should be u128")
 }
+
+func TestDebugPrint(t *testing.T) {
+	//Save the old stdout
+	rescueStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	vm := defaultVirtualMachine()
+	vm.Context.Ap = 0
+	vm.Context.Fp = 0
+
+	writeTo(vm, VM.ExecutionSegment, 0, memory.MemoryValueFromInt(10))
+	writeTo(vm, VM.ExecutionSegment, 1, memory.MemoryValueFromInt(20))
+	writeTo(vm, VM.ExecutionSegment, 2, memory.MemoryValueFromInt(30))
+
+	var startRef ApCellRef = 0
+	var endRef FpCellRef = 3
+	start := Deref{startRef}
+	end := Deref{endRef}
+	hint := DebugPrint{
+		start: start,
+		end:   end,
+	}
+	expected := []byte("[DEBUG] a\n[DEBUG] 14\n[DEBUG] 1e\n")
+	err := hint.Execute(vm)
+
+	w.Close()
+	out, _ := io.ReadAll(r)
+	fmt.Println(string(out))
+	//Restore stdout at the end of the test
+	os.Stdout = rescueStdout
+
+	require.NoError(t, err)
+	require.Equal(t, out, expected)
```

Comment:
```suggestion
        require.Equal(t, expected, out)
```
Expected goes in the right

Reply:
Fixed